src/google-classroom-extractor/edfi_google_classroom_extractor/mapping/assignments.py

```diff
@@ -68,13 +68,10 @@ def coursework_to_assignments_dfs(
                 "dueTime.minutes",
             ]
         ]
-        filled_df: Optional[DataFrame] = due_date_df.fillna("0")
-        if filled_df is None:
-            raise ValueError  # fillna will never return None in this usage
-        coursework_df["DueDateTime"] = filled_df.apply(
-            lambda date_element: datetime(
-                *to_numeric(date_element, downcast="integer")
-            ),
+        coursework_df["DueDateTime"] = due_date_df.apply(
+            lambda date_element: ""
+            if date_element.isnull().any()
```

Comment:
How quickly we forget things! For example, I forgot this lovely Python syntax of "a if (condition) else b" 😄 .

Reply:
😎